.travis.yml

```diff
@@ -19,6 +19,7 @@ script:
 - java -version
 script:
 - TERM=dumb ./gradlew assemble check --info --stacktrace
+- cd modules/minigl && TERM=dumb ../../gradlew clean test -Dtest.minigl_db_driver=postgres
```

Comment:
Perhaps you need to add `--rerun-tasks` here.

Reply:
Thanks, I've added that now